internal/rpc/jsonrpc/methods.go

```diff
@@ -2400,23 +2404,57 @@ func (s *Server) purchaseTicket(ctx context.Context, icmd interface{}) (interfac
 		expiry = int32(*cmd.Expiry)
 	}
 
-	ticketFee := w.TicketFeeIncrement()
+	dontSignTx := false
+	if cmd.DontSignTx != nil {
+		dontSignTx = *cmd.DontSignTx
+	}
 
-	// Set the ticket fee if specified.
-	if cmd.TicketFee != nil {
```

Comment:
In practice we don't need this anymore, but it is technically a breaking change to ignore it.  And unfortunately, it will have to be specified before the option to disable signing, because of positional json-rpc parameters.

I think purchaseticket has served us long enough and we should consider a new method in the next release without these pitfalls.  In the meantime, we need documentation added to this fee parameter that it is ignored.

Reply:
Returned this code part